helpers.mojo

```diff
@@ -19,22 +27,18 @@ fn find_chr_next_occurance_simd[
     Generic Function to find the next occurance of character using SIMD instruction.
     """
 
-    let t_slice = in_tensor.num_elements() - start
-    let max_iter = t_slice // simd_width
-
-    if max_iter > 0:
-        for nv in range(0, max_iter * simd_width, simd_width):
-            let simd_vec = in_tensor.simd_load[simd_width](start + nv)
-            let bool_vec = simd_vec == chr
-            if bool_vec.reduce_or():
-                for i in range(len(bool_vec)):
-                    if bool_vec[i]:
-                        return start + nv + i
-
-    for n in range(max_iter * simd_width, t_slice):
-        let simd_vec = in_tensor.simd_load[1](start + n)
-        if simd_vec == chr:
-            return start + max_iter * simd_width + n
```

Comment:
Should have been `start + n` I think.

Reply:
I think here is the reason why the function was returning garbage at the end of the tensors.